crane/crane.go

```diff
@@ -92,6 +94,19 @@ func checkDockerClient() {
 			printErrorf("Unsupported client version! Please upgrade to Docker %v or later.\n", intJoin(requiredDockerVersion, "."))
 		}
 	}
+	actualDockerVersion = versions
+}
+
+func validateDockerClientAbove(expectedVersions []int) bool {
```

Comment:
Hmm. I'm actually not against raising the minimum required Docker version quite agressively. If you're stuck on an old Docker version, you can still use an old Crane version.

What's the minimum version for `docker tag` without `force` to work as expected?